static/app/components/assigneeBadge.tsx

```diff
@@ -151,6 +151,7 @@ const StyledTag = styled(Tag)`
   }
   & > div {
     height: 24px;
+    border-style: dashed;
```

Comment:
I’m not sure if that is the right move. Fewer props is generally better, but if we have the use-case of having different border styles for our tags in the app, having a prop for that is imo preferred over custom styles overrides, especially if they rely on the dom structure of the components with `> div` selectors.

since this is the only usage, it’s probably fine, the question is more likely why does this one Tag have to be such an exception that it can’t use normal borders like every other Tag 😅

Reply:
> since this is the only usage, it’s probably fine, the question is more likely why does this one Tag have to be such an exception that it can’t use normal borders like every other Tag 😅

Good catch on updating the styling. I'm going to check if there are other instances of this.
Agree about the dashed border. I'm going to document this, and then at some point, we need a designer to strike the hammer on this :)